### DISCUSSION TITLE
In a german dict file, how can I force the use of us_en instead of the default british pronunciation?

### DISCUSSION THREAD

**Discussion by GithubAnon0000** [espeak-ng/espeak-ng#2001] [Q&A] [answered]
I am using `piper` with [thorsten voice (high german)](https://github.com/thorstenMueller/Thorsten-Voice). `piper` is a tts tool that uses `espeak-ng`s dict file to get the pronunciation right. If I want to include the word "daily" in a german sentence I'd create a de_extra file with the following contents:
```
daily	_^_EN
``` 
Then I'd compile it. The result: both `piper` as well as `espeak-ng` pronounce it correctly.
I'd like to do the same with the word `hassels` and have hassels pronounced in american english. But `espeak-ng -x "Daily Hassles" -v en-us` and `espeak-ng -x "Daily Hassles" -v en` both print `d'eIli h'as@Lz` but the pronounciation is different. I was hoping the ipa was different and I could put it into the de_extra file.

I tried the following to no avail:
```
hassels	_^_EN_US
hassels	_^_EN-US
?3 hassels	_^_EN
```
According to [en_rules](https://github.com/espeak-ng/espeak-ng/blob/master/dictsource/en_rules) the `?3` is a conditional rule. I guess it only applies when setting the language to en-us though, so it doesn't help unfortunately.

Any help is appreciated!

**Comment by hozosch** [comment, marked as answer]
I'd love this to be possible as well. Such possibility existed in the original ESpeak project, but it has been taken out, because @rhdunn stated in one of my issues that he wants to do it in a more general / robust way. Also, if that option were still there, it would mean that every word that would get the _^_en flag would be pronounced in US English, which is probably not what you had in mind.

**Reply by GithubAnon0000** [reply]
I see. I wouldn't really mind if everything is US english for this specific set of audio files. 

Since it's not currently possible to use US english in that way, I created a hack which works fine enough.
de_extra now contains the following rule:
```
hassles	hEhs@Lz
```

It sounds a little bit weird with espeaks voice but sounds close enough with thorstens voice. See my example sentence ("Erklären Sie das Modell der Daily Hassles.") [audio file.](https://github.com/user-attachments/files/17058176/OUTPUT.zip)

_sha512sum:_
```
5184c1b3ae280f0de83e7f67ae9112643ef89911d75b246a6e3fc4bde8ba73f3d7ffcf3a2955c5cc1676e93ce3861ec00488c2557527b09a42c372ab56722bf9  OUTPUT.zip
```

**Comment by hozosch** [comment]
If your goal was to make it sound right with Torsten, this may have been the better solution anyway, since the phoneme output is the same. ESpeak's just using different sounds for it in US English, and it would therefore have an impact on its own pronunciation, but not with sonata, I should imagine.

**Reply by GithubAnon0000** [reply]
If I'd use mbrola voices (`espeak-ng -x -v mb-de2 "Daily Hassles."`), could I specify us english in that case? Or does your answer from above apply?
